content/applications/finance/accounting/customer_invoices/credit_notes.rst

```diff
@@ -2,102 +2,111 @@
 Credit notes and refunds
 ========================
 
-A **credit note**, or **credit memo**, is a document issued
-to a customer that notifies them that they have been credited
-a certain amount.
+A **credit/debit note**, or **credit/debit memo**, are documents issued to a customer that notify
+them that they have been *credited/debited* a certain amount.
 
 There are several reasons that can lead to a credit note, such as:
-   * a mistake in the invoice
-   * a return of the goods, or a rejection of the services
-   * the goods delivered are damaged
+ - a mistake in the invoice;
+ - a return of the goods, or a rejection of the services;
+ - the goods delivered are damaged.
 
 .. note::
-   Issuing a credit note is the only legal way to cancel,
-   refund or modify a validated invoice. Don’t forget to
-   *register the payment* afterward if you need to send money
-   back to your customer.
+   Issuing a credit/debit note is the only legal way to cancel, refund, or modify a validated
+   invoice. Do not forget to **register the payment** afterward if you need to send money back to
+   your customer.
```

Comment:
```suggestion
   Issuing a credit/debit note is the only legal way to cancel, refund, or modify a validated
   invoice. Do not forget to **register the payment** afterward if you need to send money back to
   your customer and/or validate the **return** if a storable product is returned.
```
I was looking for an inventory doc on returns that we could link to and didn't find one, but I did find a mediocre sales doc: Process returns and refunds. I don't know if it's worth linking to though...

Reply:
It may get updated, some day. I'll link it, thanks!